src/common/devcard.ts

```diff
@@ -46,15 +52,38 @@ const getFavoriteSourcesLogos = async (
     )
     .innerJoin(
       Source,
-      'source',
-      'source.id = p."sourceId" and source.active = true and source.private = false',
+      alias,
+      `"${alias}".id = p."sourceId" and "${alias}".active = true and "${alias}".private = false`,
```

Comment:
For devCard can we not remove the private omit?
It seems "ok" to expose private squads basically.

Reply:
We'd rather not to though. Just in case